src/components/protocol_handler/src/protocol_handler_impl.cc

```diff
@@ -1106,6 +1108,24 @@ void ProtocolHandlerImpl::OnTMMessageSendFailed(
                                    << "bytes failed, connection_key "
                                    << message->connection_key()
                                    << "Error_text: " << error.text());
+
+  uint32_t connection_handle = 0;
```

Comment:
This changes also need to be in separate commit

Reply:
@LuxoftAKutsan  Done in 4b8fbb11f16569ce0079f368ae0aa95540ffc89f